samples/generics/generic_class.jakt

```diff
@@ -0,0 +1,7 @@
+class A<T> {
```

Comment:
Can you turn this into a test? You can add some comments to the top to tell the test runner how to test it. The other examples have them, so you should be able to use a similar pattern.

Reply:
Sure :)